src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,43 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.0
+
+### New features
+
```

Comment:
```suggestion
To be compliant with the [Kubernetes Multicluster Services specification][MCS KEP], service discovery now distributes a single aggregated ServiceImport to each cluster in the exported service's namespace. Previously each cluster distributed its own ServiceImport copy that was placed in the submariner namespace.
```

Reply:
... that was placed in the `submariner-operatror` namespace